Lib/enum.py

```diff
@@ -585,7 +585,6 @@ def __new__(cls, value):
                         'error in %s._missing_: returned %r instead of None or a valid member'
                         % (cls.__name__, result)
                         )
-            exc.__context__ = ve_exc
```

Comment:
I think this line (and the resulting "another exception occurred" traceback) still makes sense if we're in the case where `_missing` returned data of invalid type.

Reply:
Thanks for your comment)
Scold me if I'm wrong, but variable `ve_exc` duplicate all possible exceptions from `_missing_` and if save exception from `ve_exc` we will just duplicate exception from `_missing_`

Reply:
No, thank you taking a look at this (I'm the original reporter of the bug).

I meant that maybe the assignment should be moved into the elif above it, because it would still make sense for that case.

That way a value that's simply missing would only trigger one ValueError, but if _missing did return something but with an incorrect type then we would get a TypeError plus the indication that a suitable match was missing in the first place.

Reply:
Thanks for your explanation
This has been fixed